client/signup/steps/p2-join-workspace/style.scss

```diff
@@ -0,0 +1,204 @@
+.is-p-2-join-workspace .p2-step-wrapper {
+	max-width: 620px;
+
+	button.components-button.is-busy {
+		background-image: none;
+	}
+}
+
+.p2-join-workspace__workspace-list {
+	margin-bottom: 2.5em;
```

Comment:
I think I'm seeing the usage of `em` for margins, paddings, font-sizes, etc for the first time in Calypso codebase. Is it a new recommended practice?

Reply:
Whoops, that should have been `rem` instead. Fixed~